tools/devtool

```diff
@@ -1030,12 +1030,14 @@ cmd_build_release_archive() {
     validate_version "$version"
     say "Will start preparing release archive for v$version ..."
     get_user_confirmation || die "Aborted."
+    release_suffix="v$version-$(uname -m)"
+    archive_name="firecracker-$release_suffix.tgz"
+    archive_checksum="$archive_name.checksum.txt"
```

Comment:
should we also have the algorithm used for calculating the checksum embedded in the name?
like `$archive_name.sha256.txt`, so that users know what algorithm is used and be able to verify the binary